politeiawww/cmd/politeiavoter/sampleconfig/sampleconfig.go

```diff
@@ -43,20 +43,24 @@ const FileContents = `[Application Options]
 ; call are run over GRPC and require the wallet GRPC certificate and wallet
 ; password.
 ; wallethost=localhost
-; walletgrpccert=
+; walletgrpccert=~/.dcrwallet/rpc.cert
 ; walletpassphrase=
-; clientcert=client.pem
-; clientkey=client-key.pem
 
 ; Client certificates are required to communicate with dcrwallet. Generate a
-; client certificate key using the gencerts utility that is provided by dcrd. For
-; example on a machine with a local wallet inside the politeiavoter directory:
-; 'gencerts -L client.pem client-key.pem'
-; Then append 'client.pem' to '~.dcrwallet/clients.pem'. NOTE: the wallet looks
-; for the file called clients (plural).
-
-clientcert=client.pem
-clientkey=client-key.pem
+; client certificate key using the gencerts utility that is provided by dcrd.
+; For example, on a machine with a local wallet, client certificates can be
+; generated using the following command:
+;
+; $ gencerts ~/.politeiavoter/client{,-key}.pem
+;
+; The 'client.pem' must then be appened onto '~/.dcrwallet/clients.pem'. This
+; can be done using the following command:
+;
+; $ cat ~/.politeiavoter/client.pem >> ~/.dcrwallet/clients.pem
+;
+; NOTE: dcrwallet looks for the file called clients.pem (plural).
+clientcert=~/.politeiavoter/client.pem
+clientkey=~/.politeiavoter/client-key.pem
```

Comment:
I think these should be commented out to match sample config files for [dcrd](https://github.com/decred/dcrd/blob/8f5270b707daaa1ecf24a1ba02b3ff8a762674d3/sampleconfig/sampleconfig.go), [dcrwallet](https://github.com/decred/dcrd/blob/8f5270b707daaa1ecf24a1ba02b3ff8a762674d3/sampleconfig/sampleconfig.go) and [dcrctl](https://github.com/decred/dcrctl/blob/da63da7aa7bf71e1fd9fdb98dd83b3857e4a5f92/sample-dcrctl.conf).

Reply:
Correct, all default settings in the sample config should be commented out.  Fixed.